rest-server/src/models/job.js

```diff
@@ -416,6 +414,10 @@ class Job {
             'diskMB': 0,
           },
         },
+        "applicationCompletionPolicy": {
+          "minFailedTaskCount": 1,
+          "minSucceededTaskCount": minSucceededTaskCount
```

Comment:
Please fix [lint](https://travis-ci.org/Microsoft/pai/jobs/394413544): use single quotes, and add a trailing comma at the end.

Reply:
ok

---
In reply to: [196670819](https://github.com/Microsoft/pai/pull/742#discussion_r196670819) [](ancestors = 196670819)